sources/packages/backend/libs/utilities/src/config/config.models.ts

```diff
@@ -102,3 +102,15 @@ export interface InstitutionIntegrationConfig {
 export interface DatabaseConfiguration {
   databaseName: string;
 }
+
+export interface RedisConfiguration {
+  redisHost: string;
+  redisPort: number;
+  redisPassword: string;
+  redisStandaloneMode: boolean;
+}
+
+export interface QueueDashboardConfiguration {
+  queueDashboardUser: string;
```

Comment:
For user name and pass we also have `UserPasswordCredential` interface already used by other configs.